index.html

```diff
@@ -1,16 +1,269 @@
-<head>
-  <meta charset="utf-8">
-  <title>Redirecting</title>
-  <noscript>
-    <meta http-equiv="refresh" content="1; url=latest/" />
-  </noscript>
-  <script>
-    window.location.replace("latest/");
-  </script>
-</head>
-<body>
-  Redirecting to <a href="latest/">latest/</a>...
-</body>
-</html>
\ No newline at end of file
+
+<!doctype html>
+<html lang="en" class="no-js">
+  <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="canonical" href="https://iceoryx.io/v2.0.5/">
+      <link rel="icon" href="images/favicon.ico">
+      <meta name="generator" content="mkdocs-1.2.3, mkdocs-material-8.1.3">
+      <title>Home - iceoryx.io</title>
+      <link rel="stylesheet" href="assets/stylesheets/main.edf004c2.min.css">
+      <link rel="stylesheet" href="assets/stylesheets/palette.e6a45f82.min.css">
+      <link rel="stylesheet" href="stylesheets/extra.css">
+
+    <script>__md_scope=new URL(".",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
+  </head>
+    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
+    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
+    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
+    <label class="md-overlay" for="__drawer"></label>
+    <div data-md-component="announce">
+
+    </div>
+      <div data-md-component="outdated" hidden>
+        <aside class="md-banner md-banner--warning">
+        </aside>
+      </div>
+
+<header class="md-header" data-md-component="header">
+  <nav class="md-header__inner md-grid" aria-label="Header">
+    <div class="md-header__title" data-md-component="header-title" width="100%">
+      <div align="center">
+        <span class="md-ellipsis">
+          Eclipse iceoryx™ is an inter-process-communication middleware that enables virtually limitless data transmissions at constant time
+        </span>
+      </div>
+    </div>
+  </nav>
+</header>
+
+    <div class="md-container" data-md-component="container">
+      <main class="md-main" data-md-component="main">
+        <div class="md-main__inner md-grid" width="100%">
+          <div class="md-content" data-md-component="content"  align="center">
+            <article class="md-content__inner md-typeset" style="max-width: 1000px">
+
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
+
+
+<table align="center">
+<thead>
+<tr>
+<th><h2>iceoryx</h2></th>
+<th><h2>iceoryx2</h2></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td width="50%">
+  <ul>
+    <li>C++</li>
+    <li>True zero-copy</li>
+    <li>Central daemon</li>
+    <li>Multiple messaging patterns
```

Comment:
```suggestion
    <li>Supports multiple messaging patterns
```

Reply:
After two thumbs up I'm not so shure anymore 😅 

I think "Support" isn't the right term. Here "Offers" might be better and in the "true zero-copy" case I would either omit it or use something like "Offers/Implements/Uses". Although would prefere just plain "true zero-copy"

Reply:
It was intentional to have only buzzwords and a short overview without too much text. If someone does not understand the short version, the long version does also not help either if one in not deep in the field.

Reply:
I don't think so, e.g. the long version will set the context that this feature is about communication.

Reply:
See new screenshots :)